docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -378,23 +420,23 @@ bundling solution people are using.
 
 ```jsx
 import React from 'react';
+import Slider from '@material-ui/lab/SliderStyled';
 // webpack, parcel or else will inject the CSS into the page
 import styles from './CssModulesSlider.module.css';
-import Slider from '@material-ui/lab/SliderStyled';
 
 export default function CssModulesSlider() {
   return (
     <div>
       <Slider defaultValue={30} />
-      <Slider className={styles.slider} defaultValue={30} />
+      <Slider defaultValue={30} className={styles.slider} />
     </div>
   );
 }
 ```
 
-### Controlling priority ⚠️
+### CSS injection order ⚠️
 
-**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
+**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`, which make Material-UI take precedence over your `<style>`. To remove the need to mark your style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
```

Comment:
```suggestion
**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`, which gives Material-UI precedence over your custom styes. To remove the need to mark your style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
```